### ISSUE TITLE
optimize copies

### ISSUE THREAD

**Issue by mdvanes** [mdvanes/generator-mdworld-toolchain#2]
consolidate all fs.copy statements into one and all fs.copyTpl statements into one.

**Comment by mdvanes** [comment]
It is possible to use glob syntax to copy multiple files:

```
this.fs.copy(
  this.templatePath('files/**'),
  this.destinationPath('client'),
  { globOptions: { dot: true } }
);
```

but because some of the files need to be renamed (the dotfiles) this won't work at once. The only reason why the dotfiles should be renamed though, is probably that dotfiles are normally not copied. With the dot:true option this should be solved. So the example above can be used after renaming the _dotfiles to dotfiles.

There is one file (index.html) that is copied with copyTpl to replace a token. This must be excluded, but it should be possible with glob syntax.

**Comment by mdvanes** [comment]
Fixed in v0.0.8
